statsmodels/stats/anova.py

```diff
@@ -284,18 +284,25 @@ def anova_lm(*args, **kwargs):
     scale : float
         Estimate of variance, If None, will be estimated from the largest
         model. Default is None.
-    test : str {"F", "Chisq", "Cp"} or None
-        Test statistics to provide. Default is "F".
-    typ : str or int {"I","II","III"} or {1,2,3}
+    test : str {'F', 'Chisq', 'Cp'} or None
+        Test statistics to provide. Default is 'F'.
+    typ : str or int {'I','II','III'} or {1,2,3}
         The type of Anova test to perform. See notes.
-    robust : {None, "hc0", "hc1", "hc2", "hc3"}
+    robust : {None, 'hc0', 'hc1', 'hc2', 'hc3'}
         Use heteroscedasticity-corrected coefficient covariance matrix.
         If robust covariance is desired, it is recommended to use `hc3`.
 
     Returns
     -------
     anova : DataFrame
-    A DataFrame containing.
+        A DataFrame with columns:
+
+        - 'sum_sq': Sum of squared residuals
+        - 'df': Difference in sum of squared residuals
+        - Test Name: this column has a name that depends on the test used, for
+          example 'F'.  It contains the value of the test statistic.
+        - Pvalue: The column has a name that depends on the type of the test,
+          for example PR(>F).  Contains the p-value of the test.
```

Comment:
according to line 357 below the columns are
names = ['df_resid', 'ssr', 'df_diff', 'ss_diff', test, pr_test]
?

Reply:
e.g cel 16 in http://www.statsmodels.org/devel/examples/notebooks/generated/interactions_anova.html

Reply:
Depends on what is run.  Single anova aren't the same as multiple.